### DISCUSSION TITLE
Migrate UTM image to VMWARE .

### DISCUSSION THREAD

**Discussion by jpsanchezl10** [utmapp/UTM#5831] [General]
I want to use Parrot os on vmware but there is only an image for utm

the image contains a qcow2 file , a config.plist and efi_vars.fd

I converted the qcow2 to VMDK using:
qemu-img convert -p -f qcow2 -O vmdk 7D0EF31A-B972-4C5B-B45E-A906147D6367.qcow2 parrot.vmdk

When I start the image in VMWare I doesn't detects the installation media and can't boot.

how can I solve this?

When I start the machine on vmware I get :
Attempting to start up from :
-> Efi Vmware virtual sta hd -> no compatible bootloader found
-> same with CDROM -> no media

**Comment by mhdbaselar** [comment]
i have the same issue
